Action/vrijbrp.ZgwToVrijbrpUittrekselAction.action.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "ZgwToVrijbrpUittrekselAction",
+  "$id": "https://vrijbrp.nl/action/vrijbrp.ZgwToVrijbrpUittrekselAction.action.json",
+  "$schema": "https://json-schema.org/draft/2020-12/action",
+  "version": "0.0.2",
+  "listens": [
+    "vrijbrp.zaak.created"
+  ],
+  "throws": [],
+  "conditions": {
+    "==": [
+      {"var": "object.zaaktype.identificatie"},
+      "B0255"
+    ]
+  },
+  "class": "CommonGateway\\NaamgebruikVrijBRPBundle\\ActionHandler\\ZgwToVrijbrpUittrekselActionHandler",
+  "configuration": {
```

Comment:
location is required in the ActionHandler configuration, so it should be present or not required

Reply:
Funny, that didn't trigger in the tests

Reply:
Hmm, well that is something we should look into at some point.
Maybe it only triggers when trying to add an action with the gateway-ui?